frog/imports/api/graphs.js

```diff
@@ -3,8 +3,16 @@ import { uuid } from 'frog-utils'
  
 export const Graphs = new Mongo.Collection('graphs');
 
-export const addActivity = () =>
+export const addGraph = () => {
+  const id = uuid()
   Graphs.insert({
-    _id: uuid(),
-    created_at: new Date() 
+    _id: id,
+    name: 'untitled',
```

Comment:
Good point! I've never used default arguments in javascript so it does not come naturally for me. Done.